albumentations/core/transforms_interface.py

```diff
@@ -33,7 +34,7 @@ def to_tuple(param, low=None, bias=None):
             param = -param, +param
         else:
             param = (low, param) if low < param else (param, low)
-    elif isinstance(param, (list, tuple)):
+    elif isinstance(param, Sequence):
```

Comment:
Both, `tuple` and `omegaconf.ListConfig` pass this check, yet I wonder if `isinstance(x, collections.abc.Sequence)` is more correct way to express the contract. I used to think of typing package just as of hints, rather real types.

Reply:
In many cases them give the same results. In this case `typing.Sequence` is alias to `collections.abc.Sequence`

Reply:
I believe string is a Sequence as well. I suggest collections.MutableSequence.